codegen/apt/src/main/java/io/helidon/codegen/apt/FilerTextResourceImpl.java

```diff
@@ -76,7 +76,11 @@ public void lines(List<String> newLines) {
     public void write() {
         if (modified) {
             if (originalResource != null) {
-                originalResource.delete();
+                try {
```

Comment:
This means that it will only work with `clean compile` and will not be able to do incremental build.

Reply:
I assume deleting a resource only fails when using ECJ and `javac` wouldn't fail to delete the fail, right?